### PR TITLE
Added contribute document

### PR DESCRIPTION
**Pull request by kiranchavala**

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+use this file as a source of information when compiling the Release Notes for a new release.
+
+When adding information to the PendingReleaseNotes file make sure that you write a good and understandable description of the new feature or change which you have developed.
+
+Updating the PendingReleaseNotes file is preferably a part of the original Pull Request, but that is up to the developers' discretion.
+
+Fork the code 
+-------------
+
+In your browser, navigate to: [https://github.com/apache/cloudstack](https://github.com/apache/cloudstack)
```
Comment (rohityadavcloud) [inline CONTRIBUTING.md:39]: @kiranchavala you may want to change the name or URL used in this to cloudstack-terraform-provider

Comment (harikrishna-patnala) [inline CONTRIBUTING.md:39]:
```suggestion
In your browser, navigate to: [https://github.com/apache/cloudstack-terraform-provider](https://github.com/apache/cloudstack-terraform-provider)
```

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

**Comment by kiranchavala** [comment]
@rohityadavcloud  @borisstoyanov  @shwstppr 

Could you please review this

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+``` bash
+$ git push origin main
+$ git push origin feature_x
+```
+
+Now that the `feature_x` branch has been pushed to your GitHub repository, you can initiate the pull request.  
+
+To initiate the pull request, do the following:
+
+1. In your browser, navigate to your forked repository: [https://github.com/YOUR_ACCOUNT/cloudstack](https://github.com/YOUR_ACCOUNT/cloudstack)
```
Comment (shwstppr) [inline CONTRIBUTING.md:114]: Is it correct to point to CloudStack repo?

**Review by shwstppr — COMMENTED** [review]
@kiranchavala there are some references to CloudStack repo and confluence. Are they correct?

File: CONTRIBUTING.md (hunk 2 above)
Comment (rohityadavcloud) [inline CONTRIBUTING.md:114]:
```suggestion
1. In your browser, navigate to your forked repository: [https://github.com/YOUR_ACCOUNT/cloudstack-terraform-provider](https://github.com/YOUR_ACCOUNT/cloudstack-terraform-provider)
```

**Pull request merged**